modules/UI/UI.js

```diff
@@ -1459,4 +1436,44 @@ UI.onUserFeaturesChanged = function (user) {
     VideoLayout.onUserFeaturesChanged(user);
 };
 
+const UIListeners = new Map([
```

Comment:
Yes, definitely we should but it goes out of scope of this task. I extracted UI listeners into separate variable in order to have possibility to unsubscribe from UI Events. WDYT?